packages/address-book-controller/src/AddressBookController.test.ts

```diff
@@ -388,9 +418,198 @@ describe('AddressBookController', () => {
             memo: '',
             name: 'foo',
             addressType: undefined,
+            lastUpdatedAt: MOCK_TIMESTAMP,
           },
         },
       },
     });
   });
+
+  it('should emit contactUpdated event when adding a contact', () => {
```

Comment:
Code style: We're trying to prevent using the word `should` in test scenarios
```suggestion
  it('emits the contactUpdated event when adding a contact', () => {
```
If you could change other occurrences as well, that'd be awesome! Thanks 😄

Reply:
Also the pre-existing ones?

Reply:
I did all of them ✔️